test/adapter/json_api/relationship_test.rb

```diff
@@ -150,9 +150,23 @@ def test_relationship_with_everything
 
         private
 
-        def test_relationship(expected, params = {})
+        def test_relationship(expected, test_options = {})
           parent_serializer = AuthorSerializer.new(@author)
-          relationship = Relationship.new(parent_serializer, @serializer, nil, params)
+
+          serializable_resource_options = {} # adapter.instance_options
+
+          meta = test_options.delete(:meta)
+          options = test_options.delete(:options)
+          links = test_options.delete(:links)
+          association_serializer = @serializer
+          if association_serializer && association_serializer.object
+            association_name = association_serializer.json_key.intern
+            association = ::ActiveModel::Serializer::Association.new(association_name, association_serializer, options, links, meta)
+          else
+            association = ::ActiveModel::Serializer::Association.new(:dunno, association_serializer, options, links, meta)
+          end
```

Comment:
oh, I see what's going on now. @author and @serializer are for the whole test file, and this is just a helper method. hmm